src/gui/mainwindow.cpp

```diff
@@ -1400,6 +1401,21 @@ void MainWindow::showStatusBar(bool show)
     }
 }
 
+bool MainWindow::isFiltersSidebarVisible() const
+{
+    const QList<int> sizes = m_splitter->sizes();
+    return sizes[0] != 0;
+}
+
+void MainWindow::showFiltersSidebar(const bool show)
+{
+    if (show)
+        // Default sizes
+        m_splitter->setSizes({ 120, m_splitter->width() - 120 });
```

Comment:
Having no separate property for it we always restore it with default size... Isn't it a problem?

Reply:
https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#g-brace-enclosed-initializers
```c++
m_splitter->setSizes({120, m_splitter->width() - 120});
```
Etc...

Reply:
Is this the only thing remaining (except resolving conflicts) before merging?

Reply:
>Is this the only thing remaining (except resolving conflicts) before merging?

@glassez @Piccirello ?

Reply:
@AbeniMatteo
>Is this the only thing remaining (except resolving conflicts) before merging?

Does this mean that you would be willing to takeover/co-author?

Reply:
Ok. Should i open another PR and fork these changes?

Reply:
> Ok. Should i open another PR and fork these changes?

Please do, I opened a PR on my own repo & it added a merge commit here.......unsure of how to remove it without causing more damage.

Still a novice!